src/docs/components/tools/vendorCookieInspector/vendorCookieInspector.jsx

```diff
@@ -123,13 +123,37 @@ export default class VendorCookieDecoder extends Component {
 		this.setState({ localCookie: readCookie(VENDOR_CONSENT_COOKIE_NAME) });
 	};
 
+	readParams = () => {
+		var vendor_cookie = this.getUrlParam("string");
+		if (typeof(vendor_cookie) === "string") {
+			this.setState({ b64: vendor_cookie});
```

Comment:
This populates the text input but it doesn't parse the string. You probably want to add something to parse it after the vendor list is loaded:
```
if (this.state.b64) {	
  this.decodeB64(this.state.b64);
}
```